src/bmi_sdk/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bmi_sdk"
+version = "0.0.0"
+dependencies = [
+  "bmipy",
+  "numpy"
+]
+
+#[tool.setuptools]
```

Comment:
You can remove this guy.

Reply:
Thanks for taking a look!  I'll get this dropped.